docs/operation_semantics.md

```diff
@@ -1212,9 +1212,12 @@ A set of element-wise binary arithmetic operations is supported.
 
 <b> `Op(lhs, rhs)` </b>
 
-Where `Op` is one of `Add` (addition), `Sub` (subtraction), `Mul`
-(multiplication), `Div` (division), `Rem` (remainder), `Max` (maximum), `Min`
-(minimum), `LogicalAnd` (logical AND), or `LogicalOr` (logical OR).
+Where `Op` is one of `Add` (addition), `Atan2` (arc tangent), `Sub`
```

Comment:
Keep Add, Sub, Mul, and Div together. Don't put atan2 in the middle of them.

Reply:
Moved Atan2 (2-argument arctangent) to the end (before Complex)